### ISSUE TITLE
Can't Enable HelloHue

### ISSUE THREAD

**Issue by kmain4** [ledge74/HelloHue.bundle#9]
I can't seem to get HelloHue to enable. When I click on "enable hellohue" it says it is enabled, but it never changes to "Disable hellohue". The only error in the log is:

`WARNING (runtime:1065) - Generating a callback path for a function with no route: <function EnableHelloHueCallback at 0x7f07d85f9de8>`

That's all I have to go off of and can't find any fixes...

**Comment by bagg3rs** [comment]
Did you set the IP address of you Hue controller? That step is not very clear in instructions as far as I remember.

**Comment by kmain4** [comment]
Yes, it has a DHCP reservation. The odd thing is that HelloHue can control the lights manually via the "My Lights" section.

**Comment by Jorba123** [comment]
I have the same problem. Did you find a solution?

**Comment by jca1981** [comment]
Having same issue,using latest plexpass server and nvidia shield as client.
pastebin of log: https://pastebin.com/KRFUdi3p
